renderer.js

```diff
@@ -1574,6 +1629,7 @@ class SDFGRenderer {
 
     on_mouse_event(event, comp_x_func, comp_y_func, evtype = "other") {
         let dirty = false; // Whether to redraw at the end
+        let resort_transform = false; // Whether to re-sort at the end
```

Comment:
The name is too specialized for the vscode plugin. It should be viewport/selection changed or something similar.